tools/ansible/tasks/wsu/templates/cni.j2

```diff
@@ -0,0 +1,32 @@
+
+{
+    "cniVersion":"0.2.0",
+    "name":"OpenShiftNetwork",
+        "type":"win-overlay",
+ "capabilities": {
+        "dns": true
+    },
+    "ipam": {
+        "type": "host-local",
+        "subnet": "{{ ovn_host_subnet.stdout }}"
+    },
+        "policies":[
+            {
+                "name":"EndpointPolicy",
+                "value":{
+                    "Type":"OutBoundNAT",
+                    "ExceptionList":[
+                        "10.132.0.0/16"
```

Comment:
I don't think we can hardcode this either.

Reply:
What should be the value here?

Reply:
Please see [WINC-130](https://jira.coreos.com/browse/WINC-130)